server/src/main/java/org/elasticsearch/index/mapper/NumberFieldMapper.java

```diff
@@ -1053,6 +1120,8 @@ public static Query longRangeQuery(
             }
             return builder.apply(l, u);
         }
+
+        public abstract IndexFieldData.Builder getBuilder(String name);
```

Comment:
Maybe `getFieldDataBuilder` to distinguish it from the various other Builder classes we have floating around

Reply:
Done!